functions/src/Android Version/index.ts

```diff
@@ -12,43 +12,52 @@ main.use(bodyParser.json());
 
 export const getAndroidVersion = functions.https.onRequest(main);
 
-app.get('/supportedAndroidVersions', (request, response) => {
-    getSupportedAndroidVersions(response);
+app.post('/supportedAndroidVersions', (request, response) => {
```

Comment:
why have we changed these to POST methods? By REST convention these should be GETs

Reply:
same for all the other functions

Reply:
I can't send a body in the request if its GET, I use the body to send the user's token to check for admin privileges

Reply:
Why can't we use the authorization request header to pass that? It is almost never right to discard proper REST conventions because of something like this.

Reply:
That's a good point, I'll resolve the issue